pkg/cache/v3/cache.go

```diff
@@ -65,7 +65,7 @@ type ConfigWatcher interface {
 	//
```

Comment:
Should the comments above be updated? They seem to talk about the old channel behavior

Reply:
Oh good point, I guess we can simply everything regarding the cancellation behavior. Something like:
```go
type ConfigWatcher interface {
	// CreateWatch returns a new open watch from a non-empty request.
	// An individual consumer normally issues a single open watch by each type URL.
	//
	// Value channel produces requested resources, once they are available.
	//
	// Cancel is an optional function to release resources in the producer. If
	// provided, the consumer may call this function multiple times.
	CreateWatch(*Request, chan Response) (cancel func())

	// CreateDeltaWatch returns a new open incremental xDS watch.
	//
	// Value channel produces requested resources, or spontaneous updates in accordance
	// with the incremental xDS specification.
	//
	// Cancel is an optional function to release resources in the producer. If
	// provided, the consumer may call this function multiple times.
	CreateDeltaWatch(*DeltaRequest, stream.StreamState, chan DeltaResponse) (cancel func())
}
```